src/chart/Summary.js

```diff
@@ -157,8 +138,33 @@
                 height: this.fixedSize() ? this.minHeight_exists() ? this.minHeight() + "px" : null : "100%"
             })
         ;
+        var iconImage = null;
+        this._mainDiv.selectAll('img').remove();
+        if(this.iconImage() && this.iconImage() !== null){
+            iconImage = this.iconImage();
+        }else{
+            iconImage = row.icon;
+        }
+        if((this.iconColumn() === undefined && this.iconImage()) || this.iconColumn() === 'ImageURL'){
```

Comment:
You can't rely on the column name to work out if its an FAChar Icon or a image url.

But a heuristic of ```iconImage.indexOf("http") === 0``` might be good enough?